packaging/Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: skipper eskip docker-push
+
+SKIPPER_VERSION  ?= $(shell git rev-parse HEAD)
+SKIPPER_REGISTRY ?= registry-write.opensource.zalan.do/pathfinder
+SKIPPER_IMAGE    ?= $(SKIPPER_REGISTRY)/skipper:$(SKIPPER_VERSION)
+
+default: docker-build
+
+skipper:
+	CGO_ENABLED=0 go build github.com/zalando/skipper/cmd/skipper
+
+eskip:
+	CGO_ENABLED=0 go build github.com/zalando/skipper/cmd/eskip
+
+docker-build: skipper eskip
+	docker build -t $(SKIPPER_IMAGE) .
```

Comment:
Also create a .dockerignore file to avoid pushing unnecessary files to the docker build context (At least the packaging/Makefile and the packaging/readme.md come to mind)